Packs/FeedURLhaus/Integrations/FeedURLhaus/FeedURLhaus.py

```diff
@@ -0,0 +1,59 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+    chosen_urls = []
+    params['feed_url_to_config'] = {}
+    if 'feed_source' in params.keys():
+        sources = params['feed_source']
+        if 'Last 30 Days' in sources:
```

Comment:
get rid of duplicate code:
`url_suffixes = []
if 'Last 30 Days' in sources:
       url_suffixes.append("csv_recent")
 if 'Currently Active' in sources:
    url_suffixes.append("csv_online/")
`
and then go over the url suffixes with a loop, instead of writing the same code block twice.